libs/libarchfpga/src/physical_types.h

```diff
@@ -908,9 +910,12 @@ struct t_logical_block_type {
  *      name: name of the physical block type
  *      num_pb: maximum number of instances of this physical block type sharing one parent
  *      blif_model: the string in the blif circuit that corresponds with this pb type
+ *      model: logical model
+ *      model_sec: secondary logical model - used for pb_types matched to 2 available .latch models
  *      class_type: Special library name
  *      modes: Different modes accepted
  *      ports: I/O and clock ports
+ *      ports_sec: secondary I/O and clock ports - used for pb_types matched to 2 available .latch models
```

Comment:
Need bigger comments for who should use these, why they exist, what invariants are expected (don't exist for anything but negative edge triggered FF, expected to be copies of +ve FF for that case?).  But better yet, can we avoid this duplication?

Reply:
Added comments